server/app/routes/users.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+var router = require('express').Router();
+var db = require('../../db');
+var User = db.model('user');
+module.exports = router;
+
+router.get('/:id', function(req, res, next){
```

Comment:
Switch to using a `router.param` here to make it easier to add routes in the future